pipeline/transformer/transformer_sierra/src/main/scala/weco/pipeline/transformer/sierra/data/BibDataAsMarcRecord.scala

```diff
@@ -0,0 +1,48 @@
+package weco.pipeline.transformer.sierra.data
+
+import weco.pipeline.transformer.marc_common.models.{
+  MarcField,
+  MarcRecord,
+  MarcSubfield
+}
+import weco.sierra.models.SierraQueryOps
+import weco.sierra.models.data.SierraBibData
+import weco.sierra.models.marc.{Subfield, VarField}
+
+trait SierraMarcDataConversions {
+  import scala.language.implicitConversions
+
+  /* TODO:
+   * This implicit conversion stuff exists to temporarily implement SierraBibData
```

Comment:
I cannot remember which other project made use of `SierraBibData` necessitating it being in `wellcomecollection/scala-libs` ... but wherever it was it's [not there anymore](https://github.com/search?q=org%3Awellcomecollection+SierraBibData&type=code).

Reply:
Yes, There remains a small amount of code (to do with Items) in this part of scala-libs that is shared between Pipeline and API, but not enough to warrant the current shape of it all.